clang/lib/Sema/SemaPPC.cpp

```diff
@@ -251,14 +251,10 @@ bool SemaPPC::CheckPPCBuiltinFunctionCall(const TargetInfo &TI,
     return BuiltinPPCMMACall(TheCall, BuiltinID, Types);
 #include "clang/Basic/BuiltinsPPC.def"
   }
-<<<<<<< HEAD
 #ifdef SYCLomatic_CUSTOMIZATION
 #else
```

Comment:
do we need keep
```
return SemaRef.BuiltinConstantArgRange(TheCall, i, l, u);
```
?

Reply:
This line of code already remove in the SYCL repo.